plugins/TEMPLATE.nix

```diff
@@ -14,11 +16,11 @@ helpers.neovim-plugin.mkNeovimPlugin config {
 
   # Optionally, explicitly declare some options. You don't have to.
```

Comment:
Can we update this comment to stress that we actually prefer _not_ including options unless they are **very** important?

Reply:
We could. Have we actually settled on the philosophy going forward ?

Reply:
> Have we actually settled on the philosophy going forward ?

Not sure. Let's merge this as is for now and try to hash out the other details in another PR/issue/discussion/doc/etc